src/legacy/core_plugins/kibana/public/visualize/wizard/new_vis_modal.test.tsx

```diff
@@ -104,6 +104,19 @@ describe('NewVisModal', () => {
     });
   });
 
+  describe('filter for visualization types', () => {
+    it('should render as expected', () => {
```

Comment:
Perhaps could we update the description slightly so that if one does not know the expectation, they will have a better understanding of what the expectation is?  Perhaps something like "should render filtered results with meaningful accessibility descriptions"?

Reply:
I think if we rename this, we would want to make that test more specific. Currently we use `should render as expected` for most tests that just compare a snapshot. Also since usually if you update the snapshot you might not actually look exactly at the accessible description in there. So I would prefer leaving the name like that or writing a more specific test, that actually finds the description in there and check for it.

Reply:
Got it, thanks for explaining the description convention!  Good to know :)